### ISSUE TITLE
Improve Integration Account integration test for parallel runs

### ISSUE THREAD

**Issue by stijnmoreels** [arcus-azure/arcus.scripting#369]
**Is your feature request related to a problem? Please describe.**
Currently, some of our integration tests are not very suited to be run in parallel. They could cause false negatives this way.

**Describe the solution you'd like**
The `Create a single partner in an Integration Account succeeds` Integration Account test, for example, uses the same partner name to upload to the Integration Account; which could fail with 'already present' if another test is running. We should make sure that we generate a new name (so adapt the file) during the test.

**Comment by pim-simons** [comment]
@stijnmoreels I think we can close this right, since https://github.com/arcus-azure/arcus.scripting/pull/371 has been merged?

**Comment by stijnmoreels** [comment]
> @stijnmoreels I think we can close this right, since #371 has been merged?

We could, it's just that we may have to create another issue for possible other integration tests that also have this problem. But, yes, let's close this one. 👍
